Add Analog Theater scraper

rattboi opened the pull request:
Fixes #8

rattboi committed:
Add Analog Theater scraper

Fixes #8

demophoon commented on trello/plugins/analogtheater.py:
```diff
@@ -0,0 +1,97 @@
+        artist = artist.replace('featuring', ',')
+        artist = ','.join(artist.rsplit(' and ', 1))
+        artist = artist.split(',')
+        artist = map(string.strip, artist)
```
Strip is a method and not an attribute ;)

demophoon replied:
oh, wait, this is doing a map, ignore me.

demophoon replied:
Lets replace this with an anonymous function instead:
```python
artist = map(lambda x: x.strip(), artist)
```

This is for python 3 compatibility as string.strip doesn't exist there.

rattboi committed:
Add a few cleanups, including demophoon's comments

- Use Arrow's tz replace based on string. No need to import tzinfo then.
- Combine the string replacements to cleanup artists
- Just use list comprehension instead of map. Same crap anyways.

rattboi commented:
Changes made
